ecoscope/plotting/__init__.py

```diff
@@ -12,7 +11,6 @@
 __all__ = [
     "EcoPlotData",
     "add_seasons",
-    "collar_event_timeline",
```

Comment:
is this moved to a module now?

Reply:
I did not include it, but can do so. Question, when is it used? I haven't seen an example where it has been called.